includes/class-wc-session-handler.php

```diff
@@ -288,13 +307,31 @@ public function forget_session() {
 	/**
 	 * When a user is logged out, ensure they have a unique nonce by using the customer/session ID.
 	 *
+	 * @deprecated 5.3.0
 	 * @param int $uid User ID.
-	 * @return string
+	 * @return int|string
 	 */
 	public function nonce_user_logged_out( $uid ) {
 		return $this->has_session() && $this->_customer_id ? $this->_customer_id : $uid;
```

Comment:
Maybe add `wc_doing_it_wrong()` to help developers change to new method in the future?

Reply:
I added a `wc_deprecated_function()`.